rust/ql/lib/ext/generated/rust/lang-std.model.yml

```diff
@@ -0,0 +1,326 @@
+# THIS FILE IS AN AUTO-GENERATED MODELS AS DATA FILE. DO NOT EDIT.
+extensions:
+  - addsTo:
+      pack: codeql/rust-all
+      extensible: summaryModel
+    data:
+      - ["lang:std", "<&[u8] as crate::io::BufRead>::consume", "Argument[self].Element", "Argument[self].Reference.Reference", "value", "dfc-generated"]
```

Comment:
This is another weird edge involving references.  Based on the [description](https://doc.rust-lang.org/std/io/trait.BufRead.html#tymethod.consume) I don't think `consume` should have any taint flows.